apps/gitness/src/pages/run-pipeline-dialog/run-pipeline-form.tsx

```diff
@@ -109,14 +119,18 @@ export default function RunPipelineForm({
         requestClose()
         const executionId = response.body.number
         navigate(`${toExecutions}/${executionId}`)
-        // TODO: toast here ?
       })
       .catch(error => {
         console.error(error)
-        // TODO: error toast here ?
+        setLoading(false)
+        setErrorMessage(error.message)
       })
   }
 
+  if (errorMessage) {
+    return <>{errorMessage}</>
+  }
+
```

Comment:
maybe style the message error message as `text-destructive`?

Reply:
Used Alert component instead